src/main/java/com/minecolonies/coremod/blocks/BlockPaperwall.java

```diff
@@ -4,7 +4,6 @@
 import com.minecolonies.coremod.creativetab.ModCreativeTabs;
 import net.minecraft.block.material.MapColor;
 import net.minecraft.block.material.Material;
-import net.minecraft.block.properties.IProperty;
 import net.minecraft.block.properties.PropertyEnum;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.block.properties.IProperty'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AUselessImportCheck)